### ISSUE TITLE
Hide or cut responseBody when not successful

### ISSUE THREAD

**Issue by Exagone313** [TwinProduction/gatus#38]
When a condition is failing, Gatus prints the full response body in output. It may be useful when monitoring APIs, but it is not for a big HTML page.

My proposal is to either:
* cut the response body to print only the first X characters
* hide it completely
* add an option into the service to choose a behavior

**Comment by TwinProduction** [comment]
Fixed by 541e0264ab212916523460e9c52a1e8adfd25645

While evaluating the prospect of not retaining a service's evaluation result body after the evaluation because it provided no value besides the ability to log it in case of error, I decided to completely remove the response body appended to the logging message when an evaluation wasn't successful.

While the log itself was helpful for debugging, properly leveraging the `[BODY]` placeholder could also do the same thing without the user having to look at Gatus' logs to debug their applications. Furthermore, the body could be quite lengthy, which means that it may hinder future planned work on supporting longer result history.

**TL;DR**: The response body is no longer printed in the logs.
